pkg/kev/config/types.go

```diff
@@ -0,0 +1,52 @@
+package config
```

Comment:
Can we rename this file? - we normally reserve `types.go` to contain actual types used in a pkg.

Reply:
moved